commit 3dd8a257a18bb9be05a283b5122d54c504619fcb
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Oct 29 17:49:04 2018 +0000

    ARM: bcm2835_defconfig: Enable bcm2835-codec
    
    Enables the V4L2 M2M codec driver as a module.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2835_defconfig b/arch/arm/configs/bcm2835_defconfig
index 5a61367bb47c..75675420f28a 100644
--- a/arch/arm/configs/bcm2835_defconfig
+++ b/arch/arm/configs/bcm2835_defconfig
@@ -132,6 +132,7 @@ CONFIG_DMA_BCM2835=y
 CONFIG_STAGING=y
 CONFIG_SND_BCM2835=m
 CONFIG_VIDEO_BCM2835=m
+CONFIG_VIDEO_CODEC_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1427_3dd8a257a18bb9be05a283b5122d54c504619fcb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1427_3dd8a257a18bb9be05a283b5122d54c504619fcb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
